pom.xml

```diff
@@ -69,12 +69,15 @@
         <module>caf-logging-logback</module>
         <module>caf-logging-logback-converters</module>
         <module>caf-logging-logback-access-converters</module>
+        <module>caf-logging-serilog</module>
     </modules>
 
     <properties>
         <copyrightYear>2020</copyrightYear>
         <enforceCorrectDependencies>true</enforceCorrectDependencies>
         <logbackVersion>1.2.3</logbackVersion>
+        <maven.install.skip>true</maven.install.skip>
+        <maven.deploy.skip>true</maven.deploy.skip>
```

Comment:
These changes seem worrying.  Surely this will inherit to all modules and result in nothing being produced from this project?

Reply:
Removed